release-notes/current.yaml

```diff
@@ -7,6 +7,7 @@ breaking changes: |
   An empty TLS ALPNProtocols list is now treated as user-defined disablement of the TLS ALPN extension.
   Outlier detection (passive health check) is now disabled by default.
   refer to https://gateway.envoyproxy.io/docs/api/extension_types/#backendtrafficpolicy for working with passive health checks.
+  Envoy Gateway treats errors in calls to an extension service as fail-closed by default. The previous behavior can be enabled by setting the `failOpen` field to `true` in the extension service configuration.
```

Comment:
Maybe we can elaborate a bit here on what fail closed means. Basically, translation/connectivity errors would lead to replacement of proxy routing config with immediate responses.

Reply:
Done.